scst-scan/troubleshoot-scan.hbs.md

```diff
@@ -501,4 +501,29 @@ values.yaml:
     = expected: a map item with the key named "exports" (from .ytt/data.yaml:52)
 ```
 
-This is because the previously deprecated field of `scanning.metadataStore.url` is now completely removed. If this field is present in `tap-values.yaml` provided for the upgrade then the reconcillation will fail. Remove the field from the values file and run the upgrade command again to resolve this problem.
\ No newline at end of file
+This is because the previously deprecated field of `scanning.metadataStore.url` is now completely removed. If this field is present in `tap-values.yaml` provided for the upgrade then the reconcillation will fail. Remove the field from the values file and run the upgrade command again to resolve this problem.
+
+### <a id="scanning-restricted-pss"></a> Scanning in a cluster with restricted Kubernetes Pod Security Standards
+
+As part of compliance with restricted Kubernetes Pod Security Standards, the `securityContext` of containers and initContainers must be set. This applies to the `prepare` initContainers created by Tekton. When a pod does not meet Pod Security Standards, it will not be created and vulnerability scanning cannot proceed. You may see an error message similar to the following when describing the TaskRun:
```

Comment:
```suggestion
As part of compliance with the [restricted profile Kubernetes Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/), the `securityContext` of containers and initContainers must be set. This applies to the `prepare` initContainers created by Tekton. When a pod does not meet Pod Security Standards, it will not be created and vulnerability scanning cannot proceed. You may see an error message similar to the following when describing the TaskRun:
```